### DISCUSSION TITLE
Sign up for LinkFree

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#22] [General]
Hi,

If you want you can add your profile to [Linkfree](https://github.com/EddieHubCommunity/LinkFree) and join [EddieHubCommunity Github Organization](https://github.com/EddieHubCommunity). Both @aliraza and me are in Linkfree, so you can do it if you like too.

**Comment by TechStudent10** [comment]
maybe i'll consider but idk

**Reply by KendallDoesCoding** [reply]
> maybe i'll consider but idk

ok, its just a database like linktree but for coders, mostly.
